cidemipyme-gatsby/src/components/HoverCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Col} from 'reactstrap';
+
+const hoverCardKeyPrefix = 'hoverCard_';
```

Comment:
Usually constants are in camel case, but it is up to you, `HOVER_CARD_KEY_PREFIX`

Reply:
I will change all of them to upper case in a different PR then. Because I put them all of them like that.